### DISCUSSION TITLE
API for Blender bot?

### DISCUSSION THREAD

**Discussion by Timodhau** [facebookresearch/ParlAI#3530] [Q&A]
Hi, I am looking to use blender on a server and call it with api, is there already api integrated in it ? I did not find anything.

Regards

**Comment by stephenroller** [comment]
Quite a few people have [asked for a flask api](https://github.com/facebookresearch/ParlAI/issues?q=flask). We haven't had hours to implement this, but the community definitely wants it. I would _greatly_ appreciate if _anyone_ made a PR -- I would certainly take the time to guide it to landing.

**Reply by Jawad1347** [reply]
Could you please have it integrated with discord or other bots (like, telegram or facebook etc?)? thank you.

**Comment by khushaljethava** [comment]
I have created a script to run parlai model on the flask, If anyone can guide me with PR I can do it. Thank you.

**Comment by stephenroller** [comment]
https://github.com/facebookresearch/ParlAI/pull/4433 This ended up merging.
